File: euler_002.py
#!/Usr/bin/env python

fiba = []
fib = 0
fib2 = 1
temp = 0

while temp < 4000000:
    temp = fib + fib2
    fib = fib2
    fib2 = temp
    print(fib)

    if temp % 2 == 0:
        fiba.append(temp)

print(fiba)
print(sum(fiba))

File: C/euler002.c
#include <stdio.h>
int fib(int i)
{
    if(i == 0){
        return 0;
    }else if( i == 1){
        return 1;
    }else{
        return fib(i-1) + i;
    }
}

int main()
{
   int s = fib(1);
   printf("%d",fib(10));
   return 0;
}


